app.py

```diff
@@ -0,0 +1,89 @@
+from flask import Flask, session, render_template, request, abort, redirect, url_for
+from flask_sqlalchemy import SQLAlchemy
+from functions import get_posts, create_post, get_documents, upload_document, get_forms, upload_form
+import datetime
+import config
+import os
+
+app = Flask(__name__)
+
+app.config.from_object(os.environ['APP_SETTINGS'])
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(app)
+
+
+'''from models import Users, Posts, Documents, Forms''' 		# import datatables from postgresql database
+
+
+# API routes for interaction with frontend application
+
+@app.route('/api/posts')							# route description: GET => return all unarchived posts
+def api_post():
+	if not session.get('logged', None): abort(403)
```

Comment:
For python files, per [PEP8](https://www.python.org/dev/peps/pep-0008/) you should be using 4 spaces instead of tabs. It's really easy to configure filetype-specific settings in sublime, and you can even use it to quickly change all the tabs to 4 spaces.

Should be under Preferences > Settings - Syntax Specific. From there, you can copy the `"translate_tabs_to_spaces"` variable into `Python.sublime-settings` and set it to true.